airbrake/notifier.py

```diff
@@ -92,24 +108,27 @@ def __repr__(self):
 
     @property
     def context(self):
-        """The python, os, and app environment context."""
+        """System, application, and user data to make more sense of errors."""
         if not self._context:
-            self._context = {}
-            # python
             version = platform.python_version()
-            self._context.update({'language': 'Python %s' % version})
-            # os
             plat = platform.platform()
```

Comment:
I **think** this variable is only used on line 113, so you could do this instead:

```
'os': platform.platform(),
```